web/wp-content/themes/dctx-v2/docs/assets/js/vendor/jquery.min.js

```diff
@@ -0,0 +1,4 @@
+/*! jQuery v2.1.1 | (c) 2005, 2014 jQuery Foundation, Inc. | jquery.org/license */
```

Comment:
:warning: **Warning**: File appears to be minified and cannot be processed (*Internal.Tokenizer.Exception*).